IFC4x3/Properties/f/FeederLineType_35eHZrPkfE3fcLpjm1KsIK/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FeederLineType_35eHZrPkfE3fcLpjm1KsIK" Name="FeederLineType" UniqueId="c5a118f5-66ea-4e0e-9995-cedc01536494" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel" AccessState="readonlylocked">
```

Comment:
Consider `Object Typing`.

Reply:
There is a predefine list of value, so I think keep it like this would be better

Reply:
If you have a sort of a library of types, you could still use Object Typing and even exchange the types as a separate dataset.

Reply:
ea970ecd2756e34e162c9f2a7fd9aceedd1ac5f3